### PR TITLE
Bug 1370594: Lint and clean files

### PR DESCRIPTION
**Pull request by stephaniehobson**
Going to edit these, want them nice and shiny first.

### CODE REVIEW

File: kuma/static/js/wiki.js [hunk 1]
```diff
@@ -148,7 +154,7 @@
     /*
         Syntax highlighting scripts
     */
-    if($('article pre').length && ('querySelectorAll' in doc)) (function() {
+    if($('article pre').length && ('querySelectorAll' in doc)) {
```
Comment (stephaniehobson) [inline kuma/static/js/wiki.js]: I'm not sure if I did the right thing here. Should I have preserved the `(function(){})();`

File: kuma/static/js/wiki.js [hunk 2]
```diff
@@ -24,7 +24,9 @@
         var $subnavList = $subnav.find(' > ol');
         var minHeightFn = $('.zone-landing-header-preview-base').length ? setMinHeight : noop;
 
-        if(!$subnavList.length) return; // Exit if the subnav isn't set up properly
+        if(!$subnavList.length) {
```
Comment (schalkneethling) [inline kuma/static/js/wiki.js]: Nit: Do we want to do a space between the `if` and the open `(`?

File: kuma/static/js/wiki.js [hunk 3]
```diff
@@ -651,14 +466,18 @@
             });
 
             if(timeoutFlag) {
-                if(timer) clearTimeout(timer);
+                if(timer) {
+                    clearTimeout(timer);
+                }
             }else{
```
Comment (schalkneethling) [inline kuma/static/js/wiki.js]: Nit: Also here, spaces around the `else`

File: kuma/static/js/wiki.js (hunk 1 above)
Comment (schalkneethling) [inline kuma/static/js/wiki.js]: Yes, it looks like you are going to want to keep it. But you can nest it inside the outer if statement

```
if ($('article pre').length && ('querySelectorAll' in doc)) {
    (function() {
        ...
    })();
}
```

**Pull request merged**